include/mbedtls/rsa.h

```diff
@@ -516,7 +516,16 @@ int mbedtls_rsa_private( mbedtls_rsa_context *ctx,
  * \param input    buffer holding the data to be encrypted
  * \param output   buffer that will hold the ciphertext
  *
- * \return         0 if successful, or an \c MBEDTLS_ERR_RSA_XXX error code
+ * \deprecated     It is deprecated and discouraged to call this function
+ *                 in mode MBEDTLS_RSA_PRIVATE. Future versions of the libary
+ *                 are likely to remove the mode argument and have it implicitly
+ *                 set to MBEDTLS_RSA_PUBLIC.
+ *
+ * \note           Alternative implementations of RSA need not support
+ *                 mode being set to MBEDTLS_RSA_PRIVATE and may instead
+ *                 return MBEDTLS_ERR_RSA_UNSUPPORTED_OPERATION.
+ *
+ * \return         0 if successful, or an MBEDTLS_ERR_RSA_XXX error code
```

Comment:
Not critical, but here and in the other functions, it would have been better to keep the change in the `\return` line from the development branch, adding `\c` before `MBEDTLS_ERR_RSA_XXX`.